README.md

```diff
@@ -1 +1,198 @@
-# Review-one-of-your-past-projects-2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+ 
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: desktop version </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
```

Comment:
`## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.
- [Live Demo Link](https://kel-nana.github.io/Portfolio-finish-mobile-version/)
`

Great work on the read-me file I'd love to recommend you add a demo link 👍 

Kindly check the following link.

https://github.com/microverseinc/curriculum-transversal-skills/blob/main/deploy/lesson_deploy_github_pages.md
![giphy](https://user-images.githubusercontent.com/112465233/215265640-03c219d0-9055-4e00-a74c-ec89d98cad2e.gif)